MBBSEmu.Tests/ExportedModules/Majorbbs/cncwrd_Tests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using MBBSEmu.Memory;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class cncwrd_Tests : MajorbbsTestBase
+    {
+        private const int CNCWORD_ORDINAL = 130;
+
+        [Theory]
+        [InlineData("FIRST SECOND\0", 0, "FIRST", 6)]
+        [InlineData("FIRST SECOND\0", 6, "SECOND", 13)]
+        [InlineData("FIRST SECOND THIRD\0", 1, "IRST", 6)]
```

Comment:
how about a test that moves from SECOND -> THIRD cleanly?

Reply:
Added! 👍